api/resources/translations/messages-fr.properties

```diff
@@ -1270,42 +1346,76 @@ tasks.0.timestamp = Horodatage
 translation.add = Ajouter une traduction
 translation.key = Clé de traduction
-unique.id =
-upgrade = Mise à niveau
-upgrade.description =
-upload = 
-upload.favicon =
-upload.header.logo = 
-upload.icon = télécharger Icone
+unique.id = ID unique
+unknown.contact = Contact inconnu
+upgrade = Mise à jour
+upgrade.description = Pour mettre à nijourveau votre application vers une version, une version bêta ou une branche spécifique, il est recommandé d'effectuer d'abord une étape. Cela permet d'effectuer un travail en arrière-plan pour préparer l'installation sans interrompre les utilisateurs. Une fois la préparation terminée, cliquez sur Installer pour procéder à la mise à jour. Cette action ne peut pas être annulée, veuillez donc vous assurer que vos données ont été sauvegardées et que vos utilisateurs sont informés des temps d'arrêt.
+upload = Télécharger
+upload.favicon = Icône de favori
+upload.header.logo = Logo d'en-tête
+upload.icon = Télécharger Icône
 upload.json.forms.help = Le fichier doit contenir une liste de formulaires en JSON. Tous les formulaires existants seront supprimés.
 upload.json.forms.title = Télécharger un formulaire JSON
-upload.sms.forms =
+upload.sms.forms = Télécharger des formulaires SMS
 upload.xform.context = Contexte
 upload.xform.help = Les fichiers devraient contenir un fichier XForm et un fichier de contexte associé JSON. S'il y a un formulaire existant, il sera remplacé.
 upload.xform.meta.help = Le méta fichier au format JSON, y compris l'icône, le titre, le contexte, etc.
 upload.xform.title = Télécharger un formulaire xml
 upload.xform.xml = XML
 upload.xform.xml.help = La définition xform en XML.
 url = URL
-user.fullname.help =
+user.fullname.help = Actuellement non affiché nulle part dans l'application.
 user.password.current = Mot de passe actuel
-user.phone.help =
-user.place.help = 
-user.username.help =
+user.phone.help = Il doit s'agir d'un numéro de téléphone valide, sans tirets ni signes de ponctuation.
+user.place.help = Tous les utilisateurs doivent être associés à une Place du système pour être correctement localisés dans la hiérarchie et voir les données appropriées lorsqu'ils se connectent.
+user.username.help = C'est ce que vous utiliserez pour vous connecter à l'application.
 username.invalid = Nom d'utilisateur invalide. Les caractères autorisés sont les lettres en minuscule, les chiffres, tirets de soulignement (_) et trait d'union (-)
-username.taken =
+username.taken = Nom d'utilisateur "{{username}}" déjà pris.
+users.import.accepted_types = Only .csv file are accepted.
+users.import.cancel_upload = Cancel
+users.import.confirm_upload = Yes, upload
+users.import.confirm_upload_text = Are you sure you want to upload
+users.import.instructions = To add multiple users, please upload the appropriate spreadsheet.
+users.import.processing.title = Processing users
+users.import.processing.instructions = This process can take a long time, if you're uploading a lot of users, please be patient.
```

Comment:
Should we keep these translations empty instead?

Reply:
Good catch! I will add the missing translations.